frontend/src/static/js/v_summary.js

```diff
@@ -281,6 +329,24 @@ window.vSummary = {
       this.getDates();
       this.sortFiltered();
     },
+    processFileFormats() {
+      const selectedColors = ['#ffe119', '#4363d8', '#3cb44b', '#f58231', '#911eb4', '#46f0f0', '#f032e6',
+          '#bcf60c', '#fabebe', '#008080', '#e6beff', '#9a6324', '#fffac8', '#800000', '#aaffc3', '#808000', '#ffd8b1',
+          '#000075', '#808080'];
```

Comment:
may want to expose the mui-colors and make those into classes and use them instead. this one seems quite hacky. but can leave for another time. the exposing of mui-colors, we can create a new issue for that if needed.

Reply:
wouldn't it be better to use an object instead of class in js? i think it would be good to create a separate issue for exposing the mui-colors, this will allow discussion on how the colors can be incorporated into other potential uses as well.